rpmconf/Install/Util/utilfunc.sh

```diff
@@ -955,6 +928,25 @@ verifyUpgrade() {
    fi
 }
 
+verifyNGModulesInstalled() {
+	if [ x"$SKIP_NG_CHECK" = "xyes" ]; then
+		return
+	fi
+	NG_INSTALLED="no"
+	for i in $ZEXTRAS_PACKAGES; do
+		isInstalled $i
+		if [ x"$PKGINSTALLED" != "x" ]; then
+			NG_INSTALLED="yes"
```

Comment:
should we break loop when we encountered any NG modules installed?

Reply:
updated changes